libraries/classes/ConfigStorage/Relation.php

```diff
@@ -1610,7 +1610,7 @@ public function fixPmaTables($db, $create = true): void
         // Build a map of replacements between default table names and name built by the user
         foreach ($tablesToFeatures as $table => $feature) {
             // Empty, we can not do anything about it
-            if (empty($GLOBALS['cfg']['Server'][$feature])) {
+            if (empty($GLOBALS['cfg']['Server'][$feature]) && $GLOBALS['cfg']['Server'][$feature] !== false) {
                 continue;
             }
```

Comment:
Maybe it should be a dedicated if with a comment so it's more obvious

Reply:
Yeah, I think that too, let me add a comment.

Reply:
I think you also should do
```suggestion
isset($GLOBALS['cfg']['Server'][$feature]) && $GLOBALS['cfg']['Server'][$feature] !== false
```
It looks like some tests did catch this, because it's unset

Reply:
See: https://github.com/phpmyadmin/phpmyadmin/actions/runs/4588288866/jobs/8102408033?pr=18315#step:6:861

Reply:
The test fail looks weird. It cannot find array key "Server" even though "Server" was already being used as array key.

Reply:
For now, I have handled the case for tracking separately. Maybe I can fix this properly once I get around it. 
Isset check is not working as these variables are already set to false or empty strings by default.

Reply:
> For now, I have handled the case for tracking separately. Maybe I can fix this properly once I get around it.
Isset check is not working as these variables are already set to false or empty strings by default.

Well, they may be not isset even if everything leads you to think so
it's an old global, the content may be different. Or it's only in the tests

Reply:
Yeah that does make sense